docs/source/modules/models.rst

```diff
@@ -37,12 +37,16 @@ doctr.models.detection
 
 .. autofunction:: doctr.models.detection.linknet_resnet18
 
+.. autofunction:: doctr.models.detection.linknet_resnet18_rotation
+
 .. autofunction:: doctr.models.detection.linknet_resnet34
 
 .. autofunction:: doctr.models.detection.linknet_resnet50
 
 .. autofunction:: doctr.models.detection.db_resnet50
 
+.. autofunction:: doctr.models.detection.differentiable_binarization.pytorch.db_resnet50_rotation
```

Comment:
I needed to set here full path otherwise sphinx can't find this function due to this code: https://github.com/mindee/doctr/blob/739943a462de043e6b31af12933fc7c1991e8cb8/doctr/models/detection/differentiable_binarization/__init__.py#L3-L6

Reply:
Perhaps we should remove this for now then?

So far, we've been using tabs to have usage examples in PyTorch & TensorFlow, but we haven't yet planned to have seperate builds for pytorch and tensorflow (namely because we try to document high level features that have the same input & output signatures)

What do you think? :)

Reply:
@frgfm If possible, the `db_resnet50_rotation` should be deployed on Tensorflow as well to stick with our high level features vision, but I don't know if it requires a lot a work. :-/

Maybe we can move this line in a dedicated section in which some models are only available in one framework (I'm not aware if it's the case for other models). Then we can reintegrate it when the model is available on both framework :thinking: